src/Microsoft.Graph.Core/Requests/Middleware/RedirectHandler.cs

```diff
@@ -89,7 +89,17 @@ protected override async Task<HttpResponseMessage> SendAsync(HttpRequestMessage
                     }
 
                     // Set newRequestUri from response
-                    newRequest.RequestUri = response.Headers.Location;
+                    if (response.Headers.Location.IsAbsoluteUri)
+                    {
+                        newRequest.RequestUri = response.Headers.Location;
+                    }
+                    else
+                    {
+                        var baseAddress = newRequest.RequestUri.GetComponents(
+                            UriComponents.SchemeAndServer | UriComponents.KeepDelimiter, UriFormat.Unescaped);
+
+                        newRequest.RequestUri = new Uri(baseAddress + response.Headers.Location);
```

Comment:
This is the cleanest way I found to be doing this operation in NetStandard 1.1.

I did not take allocation or performance too much into consideration and went for the simplest implementation I could think of.

If there's a better/alternative way to append the Location header to the base address of the Request Uri, I can easily change it :)